#Header header
#Passes completed / total passes attempted
float64 passCompletionPercent
#Total yards successfully passed
int32 totalPassingYards
#Total yards successfully rushed
int32 totalRushingYards
